frontend/src/components/Contexts/LanguageContext.tsx

```diff
@@ -0,0 +1,56 @@
+import { createContext, FC, useContext, useState } from 'react'
+import { dictionaryList } from '../../language'
+
+interface ILanguageContext {
+    language: string
+    textDictionary: { [text: string]: string }
+    switchLanguage: (newLanguage: string) => void
+}
+
+interface Props {
```

Comment:
Props for what? Maybe consider more selfexplanatory name, for instance LanguageProviderProps

Reply:
Suggest keeping for consistent code between asset- and languagecontext